manifests/params.pp

```diff
@@ -113,6 +113,10 @@
   $ssl_stomp_port              = '6164'
   $ssl_verify                  = 'verify_none'
   $ssl_fail_if_no_peer_cert    = false
+  $ssl_secure_renegotiate      = true
+  $ssl_reuse_sessions          = true
+  $ssl_honor_cipher_order      = true
+  $ssl_dhfile                  = 'UNSET'
```

Comment:
Even though this matches the existing pattern, I think `undef` would be a better way to do this (and apparently putting `undef`s in `params.pp` may be an anti-pattern, so maybe just add above